documentation/modules/cruise-control/proc-configuring-deploying-cruise-control.adoc

```diff
@@ -98,6 +98,12 @@ spec:
         configMapKeyRef:
           name: cruise-control-metrics
           key: metrics-config.yml
+    apiUsers:
+      type: hashloginservice <11>
+      valueFrom: <12>
+        secretKeyRef:
+          name: cruise-control-api-users-secret
+          key: key
```

Comment:
Do we want this included here if it is for _advanced users_?

Reply:
Yeah, once merged, we can simply publish a small blog post showing how to use it with some Kafka web console (maybe referencing the old post about this).

Reply:
Removed this in the procedure doc here but left the documentation in the API docs

> Yeah, once merged, we can simply publish a small blog post showing how to use it with some Kafka web console (maybe referencing the old post about this).

Sounds reasonable to me, I also plan on updating the "Hacking Strimzi for Cruise Control UI" post with this information as well